packages/CopyInput/src/CopyInput.styles.js

```diff
@@ -9,6 +10,15 @@ export const CopyInput = styled.div`
   }
 
   button {
-    border-radius: 0 ${tokens.border.radius} ${tokens.border.radius} 0;
+    ${props =>
+      props.hasDefaultButtonBorder ? "" : `border-radius: 0 ${tokens.border.radius} ${tokens.border.radius} 0`};
```

Comment:
There's a different convention we've been following recently, up at the top of the declaration of this component you can do something like:

```js
export const CopyInput = styled.div(
  ({ hasDefaultButtonBorder }) => css`
```
... then probably just:
```js
  [data-pka-anchor='button'] {
    ${hasDefaultButtonBorder && `border-radius: 0 ${tokens.border.radius} 0 ${tokens.border.radius};`}
  }
```